ERCS/erc-7655.md

```diff
@@ -0,0 +1,113 @@
+---
+eip: 7655
+title: ERC-5189 Operation Receipts
+description: A general interface for the detection of ERC-5189 operation receipts.
```

Comment:
```suggestion
description: Detection of ERC-5189 operation receipts.
```

I'd remove the fluff (as above) and expand a bit more on what this proposal does.